symplyphysics/laws/kinematic/vector_projections_from_angle.py

```diff
@@ -0,0 +1,28 @@
+from symplyphysics import (
+    symbols, Eq, pretty, solve, Quantity, units,
+    validate_input, validate_output, expr_to_quantity, sin, cos
+)
+
+# Description
+## Most of cases might be represented in 2-dimensional space with two orthogonal axis - vertical Y and horizontal X. Any vector in this space (velocity, force etc) can be easily
+## transformed to sum of two orthogonal vectors - projections of this vector to axis with help of angle between the vector and the horizontal axis X.
+
+velocity  = symbols('velocity')
+angle = symbols('angle')
+vertical_velocity = symbols('vertical_velocity')
+horizontal_velocity = symbols('horizontal_velocity')
+
+vertical_projection = Eq(vertical_velocity, velocity * sin(angle))
+horizontal_projection = Eq(horizontal_velocity, velocity, cos(angle))
```

Comment:
I think these laws should be separated. I do not like having 2 laws in a single file.